modules/Core/pages/register.php

```diff
@@ -429,6 +429,7 @@
     'OAUTH_FLOW' => $oauth_flow,
     'OAUTH_AVAILABLE' => NamelessOAuth::getInstance()->isAvailable(),
     'OAUTH_PROVIDERS' => NamelessOAuth::getInstance()->getProvidersAvailable(),
+    'CONTINUE_WITH' => $language->get('user', 'continue_with'),
```

Comment:
I think this needs to be changed to be within each provider, with the term being `Continue with {{provider}}`, and provider getting substituted in via i18next if that makes sense - the reason being not every language will have the same structure for this sentence

for example there'd be an array map with all providers, adding in the following item (off the top of my head 😄)
```php
[
  // ... rest of provider
  'continue_with' => $language->get('user', 'continue_with_x', ['provider' => $provider->getName()])
]
```

Reply:
Ah yeah great call, always forget about this